### PR TITLE
nimmi89/c03-docker05

### PR DESCRIPTION
**Pull request by nimmi89**
# Description

Exercise: C03-Docker05

Student Name: Nimmi Varghese

> Recreating containers by pulling images from Docker Hub.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker05/nimmi89/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# C03-Docker05
+
+## Command Execution Output
+- Commands executed to run both containers:
+```
+$ docker run -itd  netizen89/exercise02:v1
```
Comment (kikobr82) [inline classes/03class/exercises/c03-docker05/nimmi89/README.md:6]: what is the `-itd` doing in this command?

Comment (nimmi89) [reply classes/03class/exercises/c03-docker05/nimmi89/README.md:6]: Running the container in an interactive terminal detached mode.

Comment (kikobr82) [reply classes/03class/exercises/c03-docker05/nimmi89/README.md:6]: yeap.. that makes sense. Have you tried to attach to this container? were you able to go inside it and run any command?

Comment (nimmi89) [reply classes/03class/exercises/c03-docker05/nimmi89/README.md:6]: In this case  docker attach <id> threw an error
$ docker attach 761263a87aad
[Wed Jul 15 11:26:16.041046 2020] [mpm_event:notice] [pid 1:tid 140080364399944] AH00492: caught SIGWINCH, shutting down gracefully
This behaviour is caused by Apache when running the container interactively, the SIGWINCH signal is passed from the host to the container, effectively signalling Apache to shut down gracefully.

docker exec will let run arbitrary commands inside an existing container.
docker exec -it <mycontainer> bash

Comment (kikobr82) [reply classes/03class/exercises/c03-docker05/nimmi89/README.md:6]: yeap..
the -dit won't work in some cases, so ideally, you'll be using either `-d` or `-it`.
Also, when you want to connect to a container, you can use the docker exec command, like:
`docker exec -it <container> bash`
Even with the httpd, you'll be able to connect to the container and execute any commands that you may need.

**Pull request merged**